examples/nlp/text_classification/text_classification_with_bert.py

```diff
@@ -107,7 +108,7 @@
 @hydra_runner(config_path="conf", config_name="text_classification_config")
 def main(cfg: DictConfig) -> None:
     logging.info(f'\nConfig Params:\n{cfg.pretty()}')
-    trainer = pl.Trainer(**cfg.trainer)
+    trainer = pl.Trainer(plugins=[NLPDDPPlugin()], **cfg.trainer)
```

Comment:
Is this script for model parallel only? Or, if it is for both, will NLPDDPPlugin() affect things for non-model parallel case?

Reply:
Non-model parallel case works as normal with NLPDDPPLugin.